dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/models/SubSnpCoreFieldsGetVariantCoordinatesTest.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.dbsnpimporter.models;
+
+import org.junit.Test;
+
+import uk.ac.ebi.eva.commons.core.models.Region;
+
+import static org.junit.Assert.*;
+
+public class SubSnpCoreFieldsGetVariantCoordinatesTest {
+
+    @Test
+    public void snpCoordinatesMustNotChange() throws Exception {
+        SubSnpCoreFields snp = new SubSnpCoreFields(1092414368L, 526595372L, 1, "NW_003104285.1", 12108029L,
+                                                    12108029L, 1, LocationType.SNP, "10", 100002924L, 100002924L,
+                                                    null, null, null, null, null, null, null, -1, null, null, null,
+                                                    -1);
+
+        assertEquals(new Region("10", 100002924L, 100002924L), snp.getVariantCoordinates());
+    }
+
+    @Test
+    public void processSNPNotInChromosome() throws Exception {
```

Comment:
This and the following methods are not part of the process, so please change the names to reflect which cases are they testing and what the outcome should be. The test case right above this one is well defined.